pkg/controller/storagecluster/initialization_reconciler_test.go

```diff
@@ -276,6 +276,7 @@ func assertExpectedResources(t assert.TestingT, reconciler ReconcileStorageClust
 		assert.Equal(t, expected[2].Provisioner, actualSc3.Provisioner)
 		assert.Equal(t, expected[2].ReclaimPolicy, actualSc3.ReclaimPolicy)
 		assert.Equal(t, expected[2].Parameters, actualSc3.Parameters)
+		assert.Equal(t, 3, len(actualSc3.Parameters))
```

Comment:
Wouldn't it be better to test if `actualSc2.Parameters["objectStoreName"]` exists? At the very least, add a comment explaining why this is being tested separately from the other parameters if it's indeed needed.